### ISSUE TITLE
live server stopped auto reloading after update to 5.5.0

### ISSUE THREAD

**Issue by dashingvampire** [ritwickdey/vscode-live-server#287]
## I'm submitting a...

```html
[x ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Current behavior

live server not reloading the browser on its own. now i have to manually reload the page.
vscode is on autosave.

## Expected behavior

i want the chrome live server to reload .

## Environment
Version: 1.31.0 (user setup)
Date: 2019-02-05
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 6.1.7601

- Live Server: 5.5.1
- Platform:  Windows 
- Visual Studio Code: 1.31

**Comment by dashingvampire** [comment]
@ritwickdey i updated it here.

**Comment by ritwickdey** [comment]
Can you share a screenshot of vscode workspace?

**Comment by dashingvampire** [comment]
![vs](https://user-images.githubusercontent.com/37540245/52638395-e973f000-2ef7-11e9-9fb7-da765a6f9807.png)

**Comment by ritwickdey** [comment]
Okay. Can you please share a screenshot of Browser console and view souce code in browser?

**Comment by ritwickdey** [comment]
and also show the .Vscode settings.json and workspace settings.json

**Comment by dashingvampire** [comment]
@ritwickdey 
![bc](https://user-images.githubusercontent.com/37540245/52638698-86cf2400-2ef8-11e9-9f41-9f20bddc35ce.JPG)
![vsc](https://user-images.githubusercontent.com/37540245/52638699-8767ba80-2ef8-11e9-9a38-c3f1eba86351.JPG)

**Comment by ritwickdey** [comment]
and settings?

**Comment by dashingvampire** [comment]
![s](https://user-images.githubusercontent.com/37540245/52638930-170d6900-2ef9-11e9-841c-1501a7e6ea7f.JPG)
same for user

**Comment by ritwickdey** [comment]
There should be few extra line of JavaScript in HTML file.. (Live server injects the code)

**Comment by dashingvampire** [comment]
> There should be few extra line of JavaScript in HTML file.. (Live server injects the code)
where?

**Comment by ritwickdey** [comment]
I think `liveServer.settings.useWebExt` is true.. 

You have to make it false. 

Search useWebExt in workspace settings or user settings

**Comment by dashingvampire** [comment]
view source code ans console are empty.
and this was all fresh folder..and files just to test after re installing vscode and live server and restarting everything many times.
![ls](https://user-images.githubusercontent.com/37540245/52639194-c2b6b900-2ef9-11e9-97e8-fcae5c618f17.JPG)

**Comment by ritwickdey** [comment]
remove the tick. and restart server

**Comment by ritwickdey** [comment]
This is not an issue. Read the settings doc. You have to remove the tick of `useWebExt` to make it false 


https://github.com/ritwickdey/vscode-live-server/blob/master/docs/settings.md

**Comment by dashingvampire** [comment]
@ritwickdey the tick was not there..i just checked it before taking the picture in case it worked with tick there.
by default it was without tick there.

**Comment by dashingvampire** [comment]
i dont know what really happened in the background. but i downloaded the chrome extension and added the file location to the server address. for now.
will be too hectic for editing all the time.

**Comment by ritwickdey** [comment]
okk. remove the trick from User and Workspace settings. And try again.

**Comment by ritwickdey** [comment]
Please don't try the browser extension now. For simple html file, you don't need the browser extension.

**Comment by ritwickdey** [comment]
Just make sure that you don't have enabled `useWebExt` from user settings and workspace settings.

**Comment by dashingvampire** [comment]
yes . did all as you in this msg. all fresh again. still same issue.

**Comment by ritwickdey** [comment]
Can you create a fresh project in Desktop. ?

I'm not really sure that's happening. may be I can help you via TEAM VIEWER

**Comment by dashingvampire** [comment]
its fine sir. i will work in other editor for now. i had some quick practices to do. i am kind of new to all this wen design world....
i will check in my laptop later. this issue is in my pc. 
thank you for help anyways,

**Comment by bhaskarunichiranjeevi** [comment]
@ritwickdey 

Great work. But I had the same issue too. I have created an open issue but no response from you. It is failing at refreshCss() and also for html and js changes I have to do save (cmd + s) 3 or more times to refresh the browser. Please check and respond ASAP.

**Comment by bhaskarunichiranjeevi** [comment]
> Can you create a fresh project in Desktop. ?
> 
> I'm not really sure that's happening. may be I can help you via TEAM VIEWER

@ritwickdey

Great work. But I had the same issue too. I have created an open issue but no response from you. It is failing at refreshCss() and also for html and js changes I have to do save (cmd + s) 3 or more times to refresh the browser. Please check and respond ASAP.

**Comment by ritwickdey** [comment]
please send the screenshot of html, browser console and view source page.

**Comment by bhaskarunichiranjeevi** [comment]
> please send the screenshot of html, browser console and view source page.

![image](https://user-images.githubusercontent.com/40071043/52685408-9c077b00-2f17-11e9-89c2-34f88980302c.png)
![image](https://user-images.githubusercontent.com/40071043/52685622-7038c500-2f18-11e9-8a89-d63a4ed553e2.png)
![image](https://user-images.githubusercontent.com/40071043/52685483-d96c0880-2f17-11e9-83d3-9aca2a8cfc71.png)
![image](https://user-images.githubusercontent.com/40071043/52685649-85adef00-2f18-11e9-95c9-aecb2d7884d5.png)

**Comment by ritwickdey** [comment]
One easy solution is that make 'fullReload' = true from vscode settings

**Comment by bhaskarunichiranjeevi** [comment]
> One easy solution is that make 'fullReload' = true from vscode settings

It is better now but still, the refresh is happening after 3 to 4 times of doing save.

**Comment by ritwickdey** [comment]
It should not be. Can open vscode dev console (view=>Toggle developer tools) and try to save one time. Did you see any changes detected message? (wait  max 10-15 sec)

**Comment by bhaskarunichiranjeevi** [comment]
> It should not be. Can open vscode dev console (view=>Toggle developer tools) and try to save one time. Did you see any changes detected message? (wait max 10-15 sec)

No, I didn't see the changes detected message after saving once

**Comment by ritwickdey** [comment]
Hi @dashingvampire, you can safly disable `fullReload`, I've got the reason of the BUG. 

Open a new Guest window of Chrome and try again (remeber to disable the `fullReload`)
